.github/actions/pr-review/pr_review.py

```
@@ -925,6 +990,130 @@ def fetch_bound_diff(repo: str, binding: PullBinding, token: str) -> str:
+        # A delta baseline is chosen from the prior authenticated review after
+        # the workflow checkouts are complete, so its commit is not one of the
+        # two shallow snapshots available locally. Use the existing immutable
+        # compare path and its explicit completeness guard for that delta.
+        return None
+    root = _local_review_root(binding.head_sha, binding.correlation)
+    if root is None:
+        raise FetchError("immutable reviewed checkout was unavailable")
+    diff_range = f"{binding.base_sha}...{binding.head_sha}"
+    if not merge_base:
+        try:
+            base_commit = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)